src/frontend/flight/handler.rs

```diff
@@ -24,8 +24,6 @@ use crate::sync::schema::SyncSchema;
 use crate::sync::writer::SeafowlDataSyncWriter;
 use crate::sync::SyncResult;
 
-pub const SEAFOWL_SYNC_CALL_MAX_ROWS: usize = 65536;
```

Comment:
Ah ok so there's no max now at all is that right?

Reply:
Yeah, that is correct. Though keep in mind that there is a slight throughput regression the more the batches grow beyond ~100K rows.